config/configuration.json

```diff
@@ -94,6 +94,7 @@
     }
   },
   "FloatingEdgeCheck": {
+    "minimum.highway.type": "raceway",
```

Comment:
Just spotted this one little thing. 

Nit: `highway.minimum` might be simpler and clearer.